Automated Smoketest: :x:
OS: centos 7.6.1810 (Linux 3.10.0-327.28.3.el7.x86_64)
 GCC:gcc (GCC) 7.3.1 20180303 (Red Hat 7.3.1-5)
Sha: 7b9d5405401905b775c02942abf2a70bc2a00eff
Number of Error(s): 5
CMake Error at cmake/Modules/DownloadLcbDep.cmake:18 (MESSAGE):
-- Configuring incomplete, errors occurred!
make[2]: *** [plugins/couchbase/CMakeFiles/libcouchbase-build.dir/build.make:62: plugins/couchbase/libcouchbase-build/lib/libcouchbase.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:15732: plugins/couchbase/CMakeFiles/libcouchbase-build.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
CMake Error at cmake/Modules/DownloadLcbDep.cmake:18 (MESSAGE):
  Failed to download
  https://packages.couchbase.com/clients/c/mock/CouchbaseMock-1.5.19.jar
Call Stack (most recent call first):
  tests/CMakeLists.txt:70 (DOWNLOAD_LCB_DEP)
-- Configuring incomplete, errors occurred!
See also /mnt/disk1/home/vamosax/smoketest/PR-13126/build/plugins/couchbase/libcouchbase-build/CMakeFiles/CMakeOutput.log.
make[2]: *** [plugins/couchbase/CMakeFiles/libcouchbase-build.dir/build.make:62: plugins/couchbase/libcouchbase-build/lib/libcouchbase.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:15732: plugins/couchbase/CMakeFiles/libcouchbase-build.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
Updating
Checking  integrity
CONFIGURATION.md               OK
examples/rdkafka_example       OK
examples/rdkafka_performance   OK
examples/rdkafka_example_cpp   OK
Checking librdkafka integrity
librdkafka.so.1                OK
librdkafka.a                   OK
Symbol visibility              OK
Checking librdkafka++ integrity
librdkafka++.so.1              OK
librdkafka++.a                 OK
Install librdkafka to /mnt/disk1/home/vamosax/smoketest/PR-13126/build/plugins/kafka
Install librdkafka++ to /mnt/disk1/home/vamosax/smoketest/PR-13126/build/plugins/kafka
[ 20%] Built target librdkafka-build
[ 20%] Linking CXX static library ../../../../lib/libaeron_client.a
[ 20%] Built target aeron_client
configure.ac:35: installing ./ar-lib
configure.ac:26: installing ./compile
configure.ac:22: installing ./config.guess
configure.ac:22: installing ./config.sub
configure.ac:21: installing ./install-sh
configure.ac:21: installing ./missing
Makefile.am: installing ./depcomp
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking whether gcc and cc understand -c and -o together... yes
[ 20%] Built target tbb
checking if gcc supports -fvisibility=hidden flag... yes
checking if gcc supports -g flag... yes
checking if gcc supports -std=gnu89 flag... yes
checking if gcc supports -pedantic flag... yes
checking if gcc supports -Wall flag... yes
checking if gcc supports -Wextra flag... yes
checking if gcc supports -Wno-unused-parameter flag... yes
checking for ar... ar
checking the archiver (ar) interface... ar
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lin
 ... ( comment is too long, 367 lines, 27370 bytes, truncated) ... 
[Build error:](https://gist.github.com/0030badf6077de6134a254286507bfad)
Build log [Build log](https://gist.github.com/HPCCSmoketest/0030badf6077de6134a254286507bfad#file-result-19-11-05-05-39-52-log)

```

--- a/system/jlib/jlog.cpp
+++ b/system/jlib/jlog.cpp
@@ -90,7 +90,7 @@ void LogMsgSysInfo::deserialize(MemoryBuffer & in)
 
 class LoggingFieldColumns
 {
-    const EnumMapping MsgFieldMap[15] =
+    const EnumMapping MsgFieldMap[16] =
     {
         { MSGFIELD_msgID,     "MsgID    " },
         { MSGFIELD_audience,  "Audience " },
@@ -106,7 +106,8 @@ class LoggingFieldColumns
         { MSGFIELD_node,      "Node                " },
         { MSGFIELD_job,       "JobID  " },
         { MSGFIELD_user,      "UserID  " },
-        { MSGFIELD_component, "Compo " }
+        { MSGFIELD_component, "Compo " },
+        { MSGFIELD_quote,     "Quoted "}
     };
     const unsigned sizeMsgFieldMap = arraysize(MsgFieldMap);
 public:

